Getting started doc modified (#6)
scalar-labs/getting-started-with-scalardb: README.md

This document contains different underlying storage/database (such as Cassandra, Cosmos DB) set up and configurations, Please follow any one of them.

> ## Install prerequisites and Configure Scalar DB

Reviewer: ```suggestion
## Run the application
```
Since this section does more than install and configure, it's weird to title like you did.

Author: Thank you for your suggestion.
Suggestion integrated.

> 

Reviewer: ```suggestion
### Install and configure
```

Author: Thank you for your suggestion.
Suggestion integrated.

> Following softwares and Scalar DB connection configuration are required to start the demonstration.

Reviewer: ```suggestion
Please install prerequisites and configure Scalar DB to start the demonstration.
```

Author: Thank you for your suggestion.
Suggestion integrated.

> * **Install prerequisites** and **Configure Scalar DB** configuration ([**scalardb.properties**](backend/QA/src/main/resources/scalardb.properties)) file based on backend database. 

Reviewer: ```suggestion
* Install prerequisites and configure Scalar DB ([**scalardb.properties**](backend/QA/src/main/resources/scalardb.properties)) file on the basis of backend database. 
```

Author: Thank you for your suggestion.
Suggestion integrated.

    * Follow [Getting Started with Scalar DB on Cassandra](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-cassandra.md) if you use Cassandra as a backend database.
    * Follow [Getting Started with Scalar DB on Cosmos DB](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-cosmosdb.md) if you use Cosmos DB as a backend database.
    * Follow [Getting Started with Scalar DB on DynamoDB](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-dynamodb.md) if you use DynamoDB as a backend database.

> * Please install [Node.js](https://nodejs.org/en/download/)

Reviewer: ```suggestion
* Install [Node.js](https://nodejs.org/en/download/)
```

Author: Thank you for your suggestion.
Suggestion integrated.
